src/main/java/uk/gov/ons/census/action/poller/CaseProcessor.java

```diff
@@ -55,18 +55,29 @@ private void executePrinterCase(CaseToProcess caseToProcess) {
 
     String routingKey = triggeredActionRule.getActionType().getHandler().getRoutingKey();
 
-    PrintFileDto printFileDto =
-        printFileDtoBuilder.buildPrintFileDto(
-            caseToProcess.getCaze(),
-            triggeredActionRule.getActionType().getPackCode(),
-            batchId,
-            triggeredActionRule.getActionType());
-    printFileDto.setBatchQuantity(batchQty);
+    int numOfMessagesToSend = 1;
 
-    rabbitTemplate.convertAndSend(outboundExchange, routingKey, printFileDto);
+    if (triggeredActionRule.getActionType().toString().equals("CE_IC03")
+        | triggeredActionRule.getActionType().toString().equals("CE_IC04")) {
```

Comment:
always use `||` and `&&` instead of `|` and `&`... unless you're doing some kind of bitwise calculation, of course

Reply:
also this needs to be rewritten to use the enumeration instead of hard-coded strings

Reply:
Addressed 👍